SQLite download link seems to be broken

We're using `wa-sqlite` in our project (a version of it) and the make process just stopped working. I've tried the `master` version of this repo and it fails with the same error:

```
mkdir -p cache/version-3.50.1
curl -LsS https://www.sqlite.org/src/tarball/sqlite.tar.gz?r=version-3.50.1 | tar -xzf - -C cache/version-3.50.1/ --strip-components=1
tar: Error opening archive: Unrecognized archive format
make: *** [Makefile:144: deps/version-3.50.1/sqlite3.c] Error 1
```

I suspect sqlite.org is no longer serving the tarball from this link (hence the unrecognized archive format). I'm investigating further, but would appreciate pointers as to how to best handle this...

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 # dependencies
 SQLITE_VERSION = version-3.50.1
-SQLITE_TARBALL_URL = https://www.sqlite.org/src/tarball/${SQLITE_VERSION}/sqlite.tar.gz
+SQLITE_TARBALL_URL = https://www.sqlite.org/src/tarball/$(SQLITE_VERSION)/sqlite.tar.gz
 
 EXTENSION_FUNCTIONS = extension-functions.c
 EXTENSION_FUNCTIONS_URL = https://www.sqlite.org/contrib/download/extension-functions.c?get=25

